snitun/multiplexer/channel.py

```diff
@@ -75,7 +75,10 @@ async def write(self, data: bytes) -> None:
             raise MultiplexerTransportClose
 
         # Create message
-        message = MultiplexerMessage(self._id, CHANNEL_FLOW_DATA, data)
+        message = tuple.__new__(
+            MultiplexerMessage,
+            (self._id, CHANNEL_FLOW_DATA, data, b""),
+        )
 
```

Comment:
There are other places we create these, but this is the only one at volume that matters